cli/sourcemaps.ts

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+import { Command, Option, OptionValues } from 'commander';
+import figlet from 'figlet';
+import fs from 'fs';
+
+const program: Command = new Command();
+
+console.log(figlet.textSync('Instabug React Native CLI'));
+
+program
+  .name('upload-sourcemaps')
+  .usage('[options]')
+  .version('1.0.0-beta1')
+  .description('A CLI for uploading source maps to Instabug dashboard.')
+  .addOption(
+    new Option('-p, --platform <value>', 'Platform')
+      .choices(['ios', 'android'])
+      .makeOptionMandatory(),
+  )
+  .requiredOption('-d, --dir <value>', 'The path of the directory including the source map file')
+  .addOption(
+    new Option('-t, --token <value>', 'Your App Token')
+      .env('INSTABUG_APP_TOKEN')
+      .makeOptionMandatory(),
+  )
+  .addOption(
+    new Option('-v, --name <value>', 'The app version name')
+      .env('INSTABUG_APP_VERSION_NAME')
+      .makeOptionMandatory(),
+  )
+  .addOption(
+    new Option('-c, --code <value>', 'The app version code')
+      .env('INSTABUG_APP_VERSION_CODE')
+      .makeOptionMandatory(),
+  )
+  .option('-l, --label <value>', "The CodePush label if it's a CodePush release")
+  .showHelpAfterError()
+  .parse(process.argv);
+
+const options: OptionValues = program.opts();
```

Comment:
**suggestion:** We can make this more type safe as follows:

```suggestion
interface UploadSourcemapsOptions {
  platform: 'android' | 'ios';
  dir: string;
  token: string;
  name: string;
  code: number;
  label: string;
}

const options = program.opts<UploadSourcemapsOptions>();
```

Reply:
Good suggestion! I just prefer a `code` type of `string` because it'll be sent to the endpoint as a string at the end.